content/en/news/releases/1.7.x/announcing-1.7/change-notes/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Change Notes
+description: Istio 1.7 release notes.
+weight: 10
+---
+
+## Traffic Management
+
+- **Added** config option `values.global.proxy.holdApplicationUntilProxyStarts`,
+which causes the sidecar injector to inject the sidecar at the start of the
+pod's container list and configures it to block the start of all other
+containers until the proxy is ready.  This option is disabled by default.
+ ([Issue #11130](https://github.com/istio/istio/issues/11130))
+
+- **Added** SDS support for Client Certificate and CA certificate used for [TLS/mTLS Origination from Egress Gateway](/docs/tasks/traffic-management/egress/egress-gateway-tls-origination-sds/) using `DestinationRule`.
+  ([Issue #14039](https://github.com/istio/istio/issues/14039))
+
+## Security
+
+- **Improved** Trust Domain Validation to validate TCP traffic as well, previously only HTTP traffic was validated. See the [design doc](https://docs.google.com/document/d/1yfYFp2Pw3-pPACYGRWQQdrEtroAunCD3fjOF9ZGtcrU/edit#heading=h.78z3fc6weip0) for details.
+([Issue #26224](https://github.com/istio/istio/issues/26224))
```

Comment:
We can remove the design doc link here. We normally don't link Google docs in release notes. Also, the github issue below already has link to the design doc.

Reply:
It was not present before, it was added in a review before this one, I'll remove it again.